config/cable.yml

```diff
@@ -1,5 +1,6 @@
 development:
-  adapter: async
+  adapter: redis
+  url: redis://localhost:6379/1
```

Comment:
Is Redis required for Turbo to work? 🤔

Reply:
As said on the [Handbook](https://turbo.hotwired.dev/handbook/streams#integration-with-server-side-frameworks):
> Turbo itself is completely backend-agnostic

So it should not be a requirement. 

Though when performing the command `rails turbo:install` from [turbo-rails](https://github.com/hotwired/turbo-rails) gem, it automatically adds these 2 lines. 

At the moment I'm only using `Turbo Frames`.

What if I simply remove these lines and study that in another PR (when working on #14)? :)